playwright-tests/create-fuels.test.ts

```diff
@@ -11,9 +11,13 @@ test.extend({
 });
 
 test('counter contract - increment function call works properly', async ({ page }) => {
+  // make a get request to localhost:4000/v1/playground to check if the node is running
+  const response = await fetch('http://localhost:4000/v1/playground');
```

Comment:
A `waitUntilReachable` method might be useful

Reply:
Oh we don't need this code anyways, this was just for debugging 😅  Removing it